repose-aggregator/components/filters/versioning-filter/src/test/java/org/openrepose/filters/versioning/domain/ConfigurationDataTest.java

```diff
@@ -45,21 +42,17 @@
  */
 public class ConfigurationDataTest {
 
-    ConfigurationData configurationData;
-    Map<String, Destination> configuredHosts;
-    Map<String, ServiceVersionMapping> configuredMappings;
-    ServiceVersionMappingList mappings;
-    ServiceVersionMapping version1, version2;
-    Node localHost;
-    DestinationEndpoint localEndpoint;
-    ReposeCluster domain;
+    private ConfigurationData configurationData;
+    private Map<String, Destination> configuredHosts;
+    private ServiceVersionMapping version1;
+    private DestinationEndpoint localEndpoint;
```

Comment:
It's my preference, certainly. I'm not sure that I changed this everyone, because this is something I see a lot of, but I didn't want to change too much that is only tangentially related to the story.

Also, since these are tests, they aren't shipped with production code so it doesn't bother me too too much.